filebeat/module/system/auth/ingest/grok-auth-messages.yml

```diff
@@ -0,0 +1,14 @@
+description: Journald Pipeline for system/auth Filebeat module
+processors:
+  - grok:
+      description: Grok specific auth messages.
+      tag: grok-specific-messages
+      field: _temp.message
+      ignore_missing: true
+      patterns:
+        - '^%{DATA:system.auth.ssh.event} %{DATA:system.auth.ssh.method} for (invalid user)?%{DATA:user.name} from %{IPORHOST:source.address} port %{NUMBER:source.port:long} ssh2(: %{GREEDYDATA:system.auth.ssh.signature})?'
+        - '^%{DATA:system.auth.ssh.event} user %{DATA:user.name} from %{IPORHOST:source.address}'
+        - '^Did not receive identification string from %{IPORHOST:system.auth.ssh.dropped_ip}'
+        - '^%{DATA:user.name} :( %{DATA:system.auth.sudo.error} ;)? TTY=%{DATA:system.auth.sudo.tty} ; PWD=%{DATA:system.auth.sudo.pwd} ; USER=%{DATA:system.auth.sudo.user} ; COMMAND=%{GREEDYDATA:system.auth.sudo.command}'
+        - '^new group: name=%{DATA:group.name}, GID=%{NUMBER:group.id}'
+        - '^new user: name=%{DATA:user.name}, UID=%{NUMBER:user.id}, GID=%{NUMBER:group.id}, home=%{DATA:system.auth.useradd.home}, shell=%{DATA:system.auth.useradd.shell}$'
```

Comment:
Sorry, I wasn't clear in my previous comment.  It isn't just the Grok patterns, lots of things are duplicated between `journald.yml` and `files.yml`.  Can `entrypoint.yml` contain all the same processing and then only call `files.yml` or `journald.yml` at the correct point to do processing that is unique to that input?

Reply:
Thanks, I thought the same while making the last changes. I don't mind working on it, however this PR needs to get in before the FF, so at the moment I rather focus on getting CI green and then work on the pipeline improvements.

CI is running now, I'll get back to it on Monday. Hopefully to just update the pipelines.

Reply:
I created a GH issue to follow up on updating the ingest pipelines.